<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema Biblioteca - Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* === Estilo general - Tema Oscuro Biblioteca === */
QMainWindow {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #1a1a2e, stop:1 #16213e);
    color: #e0e0e0;
    font-family: &quot;Segoe UI&quot;, sans-serif;
}

/* Frame principal */
#frame_login {
    background-color: rgba(30, 30, 46, 0.95);
    border-radius: 15px;
    border: 2px solid #7b1fa2;
    padding: 30px;
}

/* Frame superior */
#frame_superior {
    background-color: transparent;
    border: none;
}

/* Labels */
QLabel {
    color: #bdbdbd;
    font-weight: 500;
    font-size: 14px;
}

#label_titulo {
    color: #ffffff;
    font-size: 24px;
    font-weight: bold;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #9c27b0, stop:1 #7b1fa2);
    padding: 10px;
    border-radius: 8px;
}

/* Campos de texto */
QLineEdit {
    background-color: #2d2d2d;
    color: #e0e0e0;
    border: 2px solid #424242;
    border-radius: 8px;
    padding: 12px;
    font-size: 14px;
    selection-background-color: #7b1fa2;
}

QLineEdit:focus {
    border: 2px solid #7b1fa2;
    background-color: #333333;
}

QLineEdit:placeholder {
    color: #666666;
}

/* Botones */
QPushButton {
    border: none;
    border-radius: 8px;
    padding: 12px;
    font-weight: 600;
    font-size: 14px;
}

#boton_login {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #9c27b0, stop:1 #7b1fa2);
    color: white;
}

#boton_login:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #ab47bc, stop:1 #8e24aa);
}

#boton_login:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #7b1fa2, stop:1 #6a1b9a);
}

#boton_salir {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #ef5350, stop:1 #c62828);
    color: white;
}

#boton_salir:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #f44336, stop:1 #d32f2f);
}

#boton_salir:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #c62828, stop:1 #b71c1c);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0" columnminimumwidth="0,0,0">
    <item row="0" column="0" colspan="3">
     <widget class="QFrame" name="frame_superior">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="boton_salir">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame_login">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>400</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_titulo">
         <property name="text">
          <string>ðŸ“š LOGIN</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_usuario">
         <property name="text">
          <string>Usuario:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usuario_login">
         <property name="placeholderText">
          <string>Ingrese su usuario</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>ContraseÃ±a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="password_login">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Ingrese su contraseÃ±a</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_mensaje">
         <property name="text">
          <string>Ingrese sus credenciales</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="boton_login">
         <property name="text">
          <string>INICIAR SESIÃ“N</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
